plugins/foreman_maintain/0.2/index.md

```diff
@@ -0,0 +1,405 @@
+
+### Incremental backup
+Incremental backups can be used to only store the changes since the last backup:
+
+First take a full backup (it is applicable for all strategies `offline` will be used in the examples):
+```
+# foreman-maintain backup offline /tmp/backups
+```
+(This will create a new directory, /tmp/backups/katello-backup-YYYY-MM-DD-hh-mm-ss)
+
+Take 1st incremental backup (this will create a new directory under /tmp/backups to house the new backup, just like the full backup directory):
+
+```
+# foreman-maintain backup offline --incremental /tmp/backup/FULL_BACKUP_DIR /tmp/backups
+```
+
+Take 2nd incremental backup (again, this will create a new directory under /tmp/backups to house this second incremental backup):
+```
+# foreman-maintain backup offline --incremental /tmp/backup/FIRST_INCREMENTAL_BACKUP_DIR /tmp/backups
+```
+
+#### Branching/rebasing incremental backups
+
+Should you choose to take a new incremental backup from, say, the full backup so you don’t need too many files when restoring those backups, simply point the command to the full backup directory, and this newest backup directory will be incremental in relation to the full backup not to the 2nd incremental backup:
+
+```
+# foreman-maintain backup offline --incremental /tmp/backup/FULL_BACKUP_DIR /tmp/backups
+```
+An example with full backup on Sunday and incremental backup for all other weekdays would look like:
+
+```bash
+#!/bin/bash -e
+DESTINATION=/var/backup
+if [[ $(date +%w) == 0 ]]; then
+  foreman-maintain backup offline $DESTINATION
+else
+  LAST=$(ls -td -- $DESTINATION/*/ | head -n 1)
+  foreman-maintain backup offline --incremental "$LAST" $DESTINATION
+fi
+exit 0
+```
+
+### Skip Pulp repositories
+There may be situations in which you want to see a system without its repository information. You can skip backing up the Pulp database with the following option:
+```
+  # foreman-maintain backup <online|offline|snapshot> --skip-pulp-content /tmp/backup
+```
+`--skip-pulp-content` skips backing up /var/lib/pulp, this option is for debugging purposes or if you plan to copy /var/lib/pulp in other ways, such as rsync or shared storage. You will not have a complete backup if you use this option.
+
+### Backup with remote databases
+For local databases (PgSQL, MongoDB) we prefer to archive the whole database data directory when the service is down. With the databases located on remote servers it is not possible and we store dump of the database instead. The dump is performed when all the other Katello related services are down and the server is in the maintenance mode so the data consistency is preserved.
+
+For snapshot strategy the dumps needs to be performed during the time when the services are down and when the snapshots are usually created and it will prolong the downtime necessary for the backup.  
```

Comment:
`For the snapshot strategy,`

Reply:
`. The use of snapshotting will typically prolong the downtime necessary for a backup.`